README.md

```diff
@@ -87,3 +87,25 @@ To clean up:
 ```
 yarn run clean
 ```
+
+## Shadowsocks Block Resistance
+
+Shadowsocks used to be blocked in some countries, and because Outline uses Shadowsocks, there has been skepticism about Outline working in those countries. In fact, people have tried Outline in the past and had their servers blocked.
+
+However, since the second half of 2020 things have changed. The Outline team and Shadowsocks community made a number of improvements that strengthened Shadowsocks beyond the censor's current capabilities.
+
+As shown in the research [How China Detects and Blocks Shadowsocks](https://gfw.report/talks/imc20/en/), the censor uses active probing to detect Shadowsocks servers. The probing may be triggered by packet sniffing, but that's not how the servers are detected.
+
+Even though Shadowsocks is a standard, it leaves a lot of room for choices on how it's implemented and deployed.
+
+First of all, you **must use AEAD ciphers**. If you are using stream ciphers, you are doing it wrong. It's very easy to break your encryption and detect your server. Outline has banned all stream ciphers, since people copy old examples to set up their servers. In fact, Outline picks the cipher for you, since people don't know how to pick it. We also generate a long random secret for you, so you are not vulnerable to dictionary-based attacks.
```

Comment:
```suggestion
First of all, you **must use AEAD ciphers**. If you are using stream ciphers, your server may be exposed. It's very easy to break your encryption and detect your server. Outline has banned all stream ciphers, since people copy old examples to set up their servers. To avoid this, Outline picks the cipher for you. We also generate a long random secret for you, so you are not vulnerable to dictionary-based attacks.
```

Reply:
Updated. I want to make it clear that we are addressing a usability issues by mentioning that users don't know how to pick a cipher.